### PR TITLE
Feature/add action shellcheck

### PR DESCRIPTION
**Pull request by amd-9**
#### ✔ Checklist:

- [x] This PR has been added to [CHANGELOG.md](https://github.com/dotenv-linter/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list);
- [x] Tests for the changes have been added (for bug fixes / features);
- [x] Docs have been added / updated (for bug fixes / features).

### CODE REVIEW

File: .github/workflows/shellcheck.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: shellcheck
+        uses: reviewdog/action-shellcheck@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-review
+          fail_on_error: true
+          path: "."
```
Comment (mgrachev) [inline .github/workflows/shellcheck.yml:19]: This is the default value. Please remove this line.

File: .github/workflows/shellcheck.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+          pattern: "*.sh"
```
Comment (mgrachev) [inline .github/workflows/shellcheck.yml:20]: This is the default value. Please remove this line.

**Review by mgrachev — COMMENTED** [review]
@amd-9 👋 Thank you for your contribution! 🚀

I have left some comments. Please look at them 👀

File: .github/workflows/shellcheck.yml (hunk 2 above)
Comment (amd-9) [reply .github/workflows/shellcheck.yml]: Removed the default value

File: .github/workflows/shellcheck.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+name: CI
+on: [pull_request]
+
+env:
+    CARGO_TERM_COLOR: always
```
[the remark this answers is not in the document]
Comment (amd-9) [reply .github/workflows/shellcheck.yml]: Fixed indentation

File: .github/workflows/shellcheck.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+name: CI
```
[the remark this answers is not in the document]
Comment (amd-9) [reply .github/workflows/shellcheck.yml]: Renamed action name

**Review by mgrachev — APPROVED** [review]

**Comment by mgrachev** [comment]
@amd-9 Thank you! 👍

🙏 If it’s not difficult for you, please support the project - click on the star on GitHub ⭐️

**Pull request merged**